### ISSUE TITLE
Transpile ESM import

### ISSUE THREAD

**Issue by rtritto** [radarsu/ts-import#29]
### Steps to reproduce
- `yarn init -y`
- `yarn set version canary`
- `yarn add devalue ts-import typescript` (_devalue_ is an ESM)
- Create `test.ts` file
```ts
import 'devalue'
```
- Create `start.js` file
```js
const { loadSync } = require('ts-import')
loadSync('./test.ts')
```
- `yarn node start.js`
output:
```console
C:\test-esm\.pnp.cjs:5645
        Error.captureStackTrace(err);
              ^

Error: require() of ES Module C:\Users\<USER>\AppData\Local\Yarn\Berry\cache\devalue-npm-4.2.3-7b5bd84f96-9.zip\node_modules\devalue\index.js from C:\test-esm\.cache\ts-import\C\test-esm\test.js not supported.
Instead change the require of index.js in C:\test-esm\.cache\ts-import\C\test-esm\test.js to a dynamic import() which is available in all CommonJS modules.
    at Object.require$$0.Module._extensions..js (C:\test-esm\.pnp.cjs:5645:15)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.require$$0.Module._load (C:\test-esm\.pnp.cjs:5487:14)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (C:\test-esm\.cache\ts-import\C\test-esm\test.js:3:1)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Object.require$$0.Module._extensions..js (C:\test-esm\.pnp.cjs:5649:33)
    at Module.load (node:internal/modules/cjs/loader:1004:32) {
  code: 'ERR_REQUIRE_ESM'
}
```

**Comment by rtritto** [comment]
Related: https://github.com/microsoft/TypeScript/issues/43329

Possible related/workaround (to check): [@brillout/import](https://www.npmjs.com/package/@brillout/import)

**Comment by yunsii** [comment]
Any progress?

**Comment by rtritto** [comment]
I think no, some help is needed

**Comment by radarsu** [comment]
I'll be looking into that.

**Comment by radarsu** [comment]
For ESM packages I've released `5.0.0-beta.0` version - fully updated to ESM. My test suite is passing, but it isn't yet battle-tested so use with caution.

**Comment by rtritto** [comment]
@radarsu same error with `ts-import@5.0.0-beta.0`.

`start.js`:
```js
const { load } = require('ts-import')
load('./test.ts')
```
